javascript/components/Contact.js

```diff
@@ -45,57 +51,132 @@ export default class Contact extends Component {
       {
         name: 'name',
         value: name,
-        label: 'Name',
+        label: (
+          <FormattedMessage
+            id="components.Contact.fields.name"
+            defaultMessage="Name"
+            description="Label for the Name field in the Contact form"
```

Comment:
Just out of curiosity, why are we hard coding all of these if we have all of these mapped out in i18n/translations/definitions.json?

Reply:
The translations files are generated from these components; there's a script that parses the client JS and pulls all the internationalization components from them.